src/components/LoginButton.module.css

```diff
@@ -1,3 +1,3 @@
 .button {
-  @apply bg-light-green text-main-dark-blue font-bold py-2 px-4 rounded;
+  @apply bg-light-green text-main-dark-blue font-bold py-2 px-4 rounded w-64 text-center cursor-pointer
```

Comment:
I would rather set this to full width and use a container div in the component with a specific width